src/redux/configureStore.js

```diff
@@ -0,0 +1,13 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+  // additional reducers could be added here
```

Comment:
- [ ] it would be better if you add the category reducer here as required in the requirements 
<img width="752" alt="Screen Shot 2022-11-08 at 12 31 37" src="https://user-images.githubusercontent.com/61790587/200553761-98e14990-2792-4f15-8c73-aaee9c958e3f.png">